Nodejs quickstarter broken with node8 - fails during npm run coverage

cleme-cd-be-node-express2-test] Running shell script
+ npm run coverage

> data@0.0.0 coverage /tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test
> nyc --include="src/**/*.ts" --reporter=text --reporter=html --reporter=lcov mocha --compilers ts:ts-node/register --recursive "test/**/*-spec.ts"


/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:316
        throw new TypeError(
              ^
TypeError: Unable to require `.d.ts` file.
This is usually the result of a faulty configuration or import. Make sure there is a `.js`, `.json` or another executable extension and loader (attached before `ts-node`) available alongside `greeter.d.ts`.
    at getOutput (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:316:15)
    at /tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:336:16
    at Object.compile (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:498:11)
    at Object.m._compile (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:392:43)
    at Module._extensions..js (module.js:663:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/ts-node/src/index.ts:395:12)
    at NYC._readTranspiledSource (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/nyc/index.js:160:26)
    at NYC.addFile (/tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/nyc/index.js:144:21)
    at /tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/nyc/index.js:176:11
    at /tmp/workspace/cleme-cd/cleme-cd-be-node-express2-test/node_modules/nyc/index.js:244:5
npm ERR! code ELIFECYCLE

--- a/boilerplates/be-node-express/init.sh
+++ b/boilerplates/be-node-express/init.sh
@@ -45,3 +45,6 @@ rm $SCRIPT_DIR/files/docker/Dockerfile.bak
 echo "copy custom files & fixes from quickstart to generated project"
 cp -rv $SCRIPT_DIR/files/. .
 cp -r $SCRIPT_DIR/fix/. src/
+
+echo "adding directories 'lib' and 'docker' to nyc exlude list to fix coverage test issue"
+sed -i 's/--reporter=text/--exclude=\\"docker\\" --exclude=\\"lib\\" --reporter=text/g' package.json

